content/open-source/slsa/what-is-slsa.md

```diff
@@ -0,0 +1,83 @@
+
+The levels are described as follows:
+
+ 
+### Level 1
+**The build process must be fully scripted/automated and generate provenance.** 
+
+Designed to be easy to adopt, this level sets a foundation for working towards the subsequent SLSA levels. The build process must be fully scripted (such as through Makefile or Github Actions) to ensure that no undocumented manual additions have been added.  _Provenance_, which refers to recorded information about a piece of software’s origin, history of changes, and contributors over the course of its lifecycle, must also be generated to provide visibility into the codebase. Provenance data should include the location of the build, the materials and system it used, the type of architecture or operating system, and the steps involved in its build. 
+
+While Level 1 does not prevent tampering, fulfilling its requirements represents an important first step for securing your software supply chain. Labeling your software with this level can also help consumers make decisions about whether the software is safe enough to use for their contexts. For more information on getting started with reaching Level 1, visit [SLSA’s quick start guide](https://slsa.dev/get-started#reaching-slsa-level-1. 
+
+### Level 2
+**Requires using version control and a hosted build service that generates authenticated provenance.** 
+
+Stepping up from Level 1, Level 2 requires the use of a hosted build service like GitHub Actions, Google Cloud Build, or Travis CI rather than a developer’s local environment. It also requires the use of a version control system that tracks change history and provides immutable references to all changes, such as git, Mercurial, Subversion, or Perforce. These changes do not need to be made public, but should be attested to by an external organization that a consumer can choose to trust or not. 
+
+The stricter requirements for Level 2 help provide more protection against software tampering and enable greater levels of trust that the provenance data is accurately represented. 
+
+### Level 3 
+**The source and build platforms meet specific standards to guarantee the auditability of the source and the integrity of the provenance respectively.** 
+
+This level aims to increase trust and harden infrastructure through a variety of requirements designed to meet specific threats. The requirements are as follows:
+
+* **[Build as code](https://slsa.dev/spec/v0.1/requirements#build-as-code)**: You must be able to verifiably derive the build definition and configuration from text file definitions stored in a version control system.  
```

Comment:
I really like the use of bullets here, I think you can do something similar in the other sections to strengthen this and make it easier for readers to scan the document